### DISCUSSION TITLE
P-Patch Community Gardens in Seattle

### DISCUSSION THREAD

**Discussion by madiframpton** [jakobzhao/geog495#59] [Final Project]
I've chosen to create a map tool for users to locate and look-up various P-Patch gardens around the Seattle Area. This tool provides the user with the name, address, and the website url for the user to copy and paste into their browser to find more information. 

Map Application : https://madiframpton.github.io/GEOG495_ppatch_gardens

Repository: https://github.com/madiframpton/GEOG495_ppatch_gardens

![Screen Shot 2021-12-11 at 8 45 24 PM](https://user-images.githubusercontent.com/91759168/145700612-c4986a5b-adfd-457d-a8c9-ad26830c02dc.png)

**Comment by ghost** [comment]
I clicked on your project because I love p-patches, I have a couple near me. I've always been interesting in seeing more community gardens in a shift towards sustainability. Cool map!

Very clean layout and I like your personal base-map, it fits thematically. The critiques I have are: The text obviously overflows in the pop up panel, maybe it could be a stagnant url somewhere else or just removed; Personally, I think the neighborhood name as the title first, then put the address underneath is easier on the eyes to look at, then you could see your neighborhood easier if you're looking for it.

**Comment by TheAdonalsium** [comment]
I do like the project-Gardens are a good choice for something to visit, so a very useful application.

For critiques, I would say that the text overflow is clearly a problem, maybe just have a message with the actual link attached instead of just the link. Additionally, the top text being the same for all the points is somewhat redundant, so you could remove that and lose nothing.
